src/devtools/client/debugger/src/reducers/pause.ts

```diff
@@ -183,12 +191,15 @@ const pauseSlice = createSlice({
     pauseHistoryIncremented(state) {
       state.pauseHistoryIndex++;
     },
-    pauseCreationFailed(state, action: PayloadAction<string>) {
-      const executionPoint = action.payload;
+    pauseCreationFailed(state) {
+      const lastPause = state.pauseHistory[state.pauseHistoryIndex];
       Object.assign(state, {
-        id: undefined,
+        seekState: "paused",
```

Comment:
Is this the right state after pause failure?

Reply:
We're going back to the last successfully created pause (`state.pauseHistory` only contains successfully created pauses, i.e. every entry has a `pauseId`). I thought that was the right thing to do since we don't really handle failed pauses anywhere in the UI, so staying at a failed pause may lead to bugs. But perhaps this (staying at the failed pause) is what we want anyway and we should just start fixing any bugs we may see with that?

Reply:
I think that's probably right, yeah. Would be confusing at best to have an execution point and pause that aren't in sync, and maybe could cause command failures if we used the pause?